examples/build.gradle

```diff
@@ -0,0 +1,121 @@
+plugins {
+    id 'java'
+}
+
+
+repositories {
```

Comment:
If we're going to have a subprojects block in the top level build.gradle then we may as well stick this there.